lib/file-info-view.coffee

```diff
@@ -10,10 +12,29 @@ class FileInfoView extends HTMLElement
     @bufferModified.classList.add('buffer-modified')
     @appendChild(@bufferModified)
 
+    @newTooltip()
+
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
+    clickHandler = ->
+      atom.clipboard.write(@getActiveItem()?.getPath?() or 'untitled')
```

Comment:
What about using `@getActiveItem()?.getPath?() or @currentPath.textContent`? The problem with constant `"untitled"` is that we are expecting everything without a path to have such name always and forever. The method `updatePathText` is written in generic way and it feels like a good fallback as it will provide us whatever is shown in the status bar if the path doesn't exist.